Makefile

```diff
@@ -21,13 +28,15 @@ ifdef E2E_TEST_EXTRA_ARGS
 	E2E_TEST_ARGS +=  $(E2E_TEST_EXTRA_ARGS)
 endif
 
+GINKGO?= build/_output/bin/ginkgo
 OPERATOR_SDK ?= build/_output/bin/operator-sdk
 GITHUB_RELEASE ?= build/_output/bin/github-release
 LOCAL_REGISTRY ?= registry:5000
 
 export KUBEVIRT_PROVIDER ?= k8s-1.13.3
 export KUBEVIRT_NUM_NODES ?= 1
 export KUBEVIRT_NUM_SECONDARY_NICS ?= 1
+export NODE_NAME=node01
```

Comment:
where do we use it? which Node's name is this? first node? node used for some purpose? what about multi-node?

Reply:
This is just used just by unit tests, let me move there.